src/Codec/CodecRegistry.php

```diff
@@ -21,13 +29,21 @@ public function offsetGet($offset)
         return $this->codecs[$offset];
     }
 
+    /**
+     * @return $this
```

Comment:
This should be probably be changed to `@return void`, as `offsetSet` will have a return type `: void` in the next PHP major release, but this change can break BC for `jaeger-client-php`